amplify-tools-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'groovy'
+apply plugin: 'idea'
+apply plugin: 'maven-publish'
+
+apply plugin: 'com.android.library'
+
+sourceSets.main.groovy.srcDirs = ["src/main/java", "src/main/groovy"]
```

Comment:
I think `sourceSets.main.groovy.srcDirs` should only be `["src/main/groovy"]`. If you have groovy sources under `src/main/java`, can you move them to the groovy dir?